crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -57,6 +60,11 @@ pub struct ConductorConfig {
 
     /// Config options for the network module. Optional.
     pub network: Option<holochain_p2p::kitsune_p2p::KitsuneP2pConfig>,
+
+    /// Config used for debugging and diagnosing issues with Holochain.
+    /// Best to not touch this unless you know what you're doing!
+    /// NB: this part of the config is **not** guaranteed to be stable.
+    pub dev: Option<DevConfig>,
```

Comment:
This is a whole new section of ConductorConfig, where we can stuff in tweaks to system-level timeouts and such. Kind of like `tuning_params`